src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogConnectionFormDataProvider.php

```diff
@@ -77,6 +83,20 @@ protected function prepareCompanyBusinessUnitChoices(): array
         return $companyBusinessUnitChoices;
     }
 
+    /**
+     * @return array [connectionFormat => FormTypeInterface]
+     */
+    protected function prepareConnectionFormatSubForms(): array
+    {
+        $connectionFormatSubForms = [];
```

Comment:
As for me, a method can be smaller. Please get rid of line 92 (to not waste lines).

Reply:
Prefer to have an empty line before structures and variables to have a better readability, since we do not have any convention here